gradle/libs.versions.toml

```diff
@@ -1,13 +1,13 @@
 [versions]
-agp = "8.3.2"
```

Comment:
Gdy próbuję odpalić apkę na tym branchu, to dostaję error:
```
The project is using an incompatible version (AGP 8.4.0) of the Android Gradle plugin. Latest supported version is AGP 8.3.2
```

Czy u Ciebie to działa, a jeśli tak to czy coś musiałeś zrobić?

Reply:
Gdy zmieniłem tylko tę linię, to wszystko mi działa